### PR TITLE
feature: asset cdn

### PR DESCRIPTION
**Pull request by sush1lemon**
- use different url for assets

### CODE REVIEW

**Comment by BigmanGHark** [comment]
We probably need to handle this in our crossdomain in future (although maybe its lax enough it wont matter), other than that looks good

**Review by BigmanGHark — APPROVED** [review]

File: client/scripts/com/monsters/display/ImageCache.as [hunk 1]
```diff
@@ -208,7 +209,7 @@ package com.monsters.display
          var l:Loadable = queue;
          l.loadState = LOADING;
          req_str = l.shouldPrepend ? prependImagePath + l.key : l.key;
-         l.loader.load(new URLRequest(req_str));
+         l.loader.load(new URLRequest(req_str), new LoaderContext(true));
```
Comment (BigmanGHark) [inline client/scripts/com/monsters/display/ImageCache.as:212]: Out of curiosity whats this do?

Comment (sush1lemon) [reply client/scripts/com/monsters/display/ImageCache.as:212]: it's for the crossdomain.xml, just telling the load function to the get crossdomain.xml of the cdn url. Like in the [AuthForm](https://github.com/bym-refitted/backyard-monsters-refitted/blob/9c0227998db95b4bbba5e517e4bd0c9947e73d5f/client/scripts/AuthForm.as#L138)

Comment (BigmanGHark) [reply client/scripts/com/monsters/display/ImageCache.as:212]: Nice one thank you

**Comment by BigmanGHark** [comment]
@React1-X Could we merge this one. This seems like all the client side changes we need for now and we can update the server url later

**Comment by React1-X** [comment]
> @React1-X Could we merge this one. This seems like all the client side changes we need for now and we can update the server url later

Yeah sure, sounds good to me. Merging.

**Pull request merged**